src/pod-security-policy/users/src_test.rego

```diff
@@ -213,7 +213,22 @@ test_user_input_container_run_as_rule_non_root_ignore_ranges {
   results := violation with input as input
   count(results) == 0
 }
-
+test_user_input_container_run_as_rule_non_root_runasnonroot_true {
+  input := {
+    "review": review(runAsNonRoot(true), [ctr("cont1", null)], null),
+    "parameters": runAsUser(rule("MustRunAsNonRoot", null))
+  }
+  results := violation with input as input
+  count(results) == 0
+}
+test_user_input_container_run_as_rule_non_root_runasnonroot_false {
```

Comment:
I think you'd get an unexpected no violations if you added a test:

```
test_user_input_container_must_run_as_rule_runasnonroot_true {
  input := {
    "review": review(runAsNonRoot(true), [ctr("cont1", null)], null),
    "parameters": runAsUser(rule("MustRunAs", [range(100, 200)]))
  }
  results := violation with input as input
  count(results) == 1
}
```

In that scenario a user would expect a violation b/c they are requiring a specific user ID range.

Reply:
Added test:
```rego
test_user_one_container_run_in_range_seccont_runasnonroot {
  input := { "review": review(null, [ctr("cont1", runAsNonRoot(true))], null), "parameters": user_mustrunas_100_200 }
  results := violation with input as input
  count(results) == 1
}
```